### ISSUE TITLE
[Android] Dragon Ball Z: Kakarot crashing issue 

### ISSUE THREAD

**Issue by zZardd** [yuzu-emu/yuzu#12662]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

android-191

### Description of Issue

Playing a part on the story mode crashes the game when starting the fight with Goku & Gohan. The fps will become 0.0 and then it will crash.

### Expected Behavior

It shouldn't crash, because if its about the world/arena bug, then it supposed to crashed when the first encounter to it using Vegeta and Trunks. That part went stable though.

### Reproduction Steps

https://github.com/yuzu-emu/yuzu/assets/108114549/86850801-d8e1-46d9-ad19-74772db1f3e3

- starting with that part, it will start but as you play a bit, it will stop and crash.

Save file:
[010051C0134F8000.zip](https://github.com/yuzu-emu/yuzu/files/13927233/010051C0134F8000.zip)

### Log File

[yuzu_log.txt.old (4).txt](https://github.com/yuzu-emu/yuzu/files/13927239/yuzu_log.txt.old.4.txt)

### System Configuration

CPU: Snapdragon 778g
GPU/Driver: Adreno 642L
RAM: 8GB
OS: Android 14

**Comment by liamwhite** [comment]
Is this a new issue or did it exist in older builds?

**Comment by zZardd** [comment]
> Is this a new issue or did it exist in older builds?

It happened since v188 up to the latest v191

**Comment by Raistlin43** [comment]
> Is this a new issue or did it exist in older builds?

It also happens to me sometimes on the loading screen from 188. And then there is the problem that every minute the fps drops to 0 and then recovers which makes it unplayable.... Rotating the mobile is NOT the solution as not all people can or have the knowledge to do it 😔.

**Comment by liamwhite** [comment]
And bitching about rooting not being a solution to the stutters is a non-starter because that's the only way we can fix it. The default kernel limits are too low. This is off-topic for this issue.

**Comment by liamwhite** [comment]
@zZardd 188 is the first build it happens on, or was it happening before that?

**Comment by zZardd** [comment]
> @zZardd 188 is the first build it happens on, or was it happening before that?

Yeah. I started playing that game on 188, but when I got to that part and started getting that issue, I tried to try the other versions to see if it will have the same issue, and it did.

**Comment by rospec14** [comment]
> Root the mobile is NOT the solution as not all people can or have the knowledge to do it 😔.

ROOT is the solution, for now anyways. If you don't want to research it, then that's your problem.
